mp/src/game/client/momentum/ui/MapSelection/BaseMapsPage.cpp

```diff
@@ -484,7 +546,7 @@ void CBaseMapsPage::GetNewMapList()
     g_pMapCache->GetMapList(vecMaps, GetMapListType());
 
     FOR_EACH_VEC(vecMaps, i)
-        AddMapToList(vecMaps[i]);
+    AddMapToList(vecMaps[i]);
```

Comment:
Can you actually wrap this in braces? It's a for loop inside a macro so visual studio wrongly formatted this.


```
FOR_EACH_VEC(vecMaps, i)
{
    AddMapToList(vecMaps[i]);
}
```